src/v/cloud_storage/tests/remote_segment_test.cc

```diff
@@ -83,8 +83,11 @@ FIXTURE_TEST(
       .delta_offset = model::offset(0),
       .ntp_revision = segment_ntp_revision};
     auto path = m.generate_segment_path(key, meta);
+    auto is_leader = should_continue_operation{
+      "lost leadership", []() { return true; }};
```

Comment:
nit: maybe rename them to `auto always = should...` or somesuch

Also maybe change the string to "no-op" or "unused" or something. It's mildly confusing as is. These continuations don't have anything to do with leadership.

Reply:
thanks, fixed